contracts/core/SedaCoreV1.sol

```diff
@@ -65,11 +75,35 @@ contract SedaCoreV1 is ISedaCore, RequestHandlerBase, ResultHandlerBase, UUPSUpg
     /// @dev Overrides the base implementation to also add the request ID and timestamp to storage
     function postRequest(
         SedaDataTypes.RequestInputs calldata inputs
-    ) public override(RequestHandlerBase, IRequestHandler) returns (bytes32) {
-        bytes32 requestId = super.postRequest(inputs);
+    ) public payable override(RequestHandlerBase, IRequestHandler) returns (bytes32) {
+        return postRequest(inputs, 0, 0, 0);
+    }
+
+    function postRequest(
+        SedaDataTypes.RequestInputs calldata inputs,
+        uint256 requestFee,
```

Comment:
Nothing to fix here but we should really have a wrapper or something to simplify the 3 fees and the rest of the inputs properties. (even if the user is overpaying, they'll get it refunded.)